crates/ruff_linter/src/rules/isort/categorize.rs

```diff
@@ -378,3 +380,20 @@ impl KnownModules {
         user_defined
     }
 }
+
+impl fmt::Display for KnownModules {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
Nit: Same as above. A traditional for loop can help to avoid the inner `format!` call (which allocates a string, if Rust isn't clever enough to optimise it away)